contexts/SubscriptionContext/tests/Integration/UseCases/AddSubscription/AddSubscriptionUseCaseTest.php

```diff
@@ -139,4 +141,19 @@ public function testGivenDataThatNeedsToBeModerated_requestNotBeMailed() {
 		$useCase->addSubscription( $request );
 	}
 
+	public function testNewMailAddressFromSubscription(): void {
+		$useCase = new AddSubscriptionUseCase( $this->repo, $this->validator, $this->mailer );
+		$class = new ReflectionClass( AddSubscriptionUseCase::class );
```

Comment:
I'd have gone for giving the UC a request model and seeing if the mailer gets invoked with the correct email object